sbpack/noncwl/utils.py

```diff
@@ -121,8 +150,11 @@ def zip_and_push_to_sb(api, workflow_path, project_id, folder_name):
         logger.error(f"File size too big: {os.path.getsize(zip_path)}")
         raise FileExistsError  # Add the right error
 
-    folder_found = list(api.files.query(project=project_id, names=[folder_name],
-                                        limit=100).all())
+    folder_found = list(api.files.query(
+        project=project_id,
+        names=[folder_name],
+        limit=100,
```

Comment:
No point in using both "limit" and "all()" like this. Let the API client determine the appropriate page size, just use "all()".